lib/model.d.ts

```diff
@@ -226,6 +268,7 @@ export type WhereValue =
   | number // literal value
   | boolean // literal value
   | null
+  | Op
```

Comment:
this tells TS to accept an object with all the operators as properties, but an `Op` is not an object, it's a Symbol itself.